### PR TITLE
Error default retention filter

### PR DESCRIPTION
**Pull request by dussault-antoine**
### What does this PR do?

Add documentation for this new retention filter

### Motivation

New default retention filter to index errors

### Preview

https://docs-staging.datadoghq.com/antoine.dussault/errors_retention_filter/tracing/trace_retention

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/trace_retention/_index.md [hunk 1]
```diff
@@ -16,30 +16,34 @@ further_reading:
...
 
 To track or monitor your volume of ingested and indexed data, see the [Usage Metrics][2] documentation.
 
 ## Retention filters
 
-After spans have been ingested by Datadog, some are kept for 15 days according to the retention filters that have been set on your account. By default, the only retention filter enabled is the [Intelligent Retention Filter](#datadog-intelligent-retention-filter), which retains error traces and traces from different latency distributions.
+After spans have been ingested by Datadog, some are kept for 15 days according to the retention filters that have been set on your account.
+
+By default, two retention filters are enabled for your account:
+- The `Error Default` retention filter indexes error spans with `status:error`. The retention rate and the query can be configured. For instance to only capture production errors, set the query to `status:error, env:production`. Disable the retention filter if you do not want to capture the errors by default.
```
Comment (urseberry) [inline content/en/tracing/trace_retention/_index.md:28]:
```suggestion
- The `Error Default` retention filter indexes error spans with `status:error`. The retention rate and the query can be configured. For instance, to capture production errors, set the query to `status:error, env:production`. Disable the retention filter if you do not want to capture the errors by default.
```

File: content/en/tracing/trace_retention/_index.md [hunk 2]
```diff
@@ -69,21 +73,24 @@ For 30 days, intelligent retention retains:
...
 
-If there are specific tags, facets, or groups of traces that you want to investigate _in detail_, meaning you want to retain more than what Intelligent Retention retains, then [create your own retention filter](#create-your-own-retention-filter). For example, you might want to keep more than a representative selection of errors from your production environment. To ensure _all_ production errors are retained and available for search and analytics for 15 days, create a 100 percent retention filter scoped to `env:prod` and `status:error`. As discussed below, this may have an impact on your bill.
+Spans indexed by the intelligent retention filter are **not counted towards the usage** of indexed spans, and hence **do not impact your bill**.
+
+If there are specific tags, facets, or groups of traces that you want to investigate _in detail_, meaning you want to retain more than what Intelligent Retention retains, then [create your own retention filter](#create-your-own-retention-filter).
```
Comment (urseberry) [inline content/en/tracing/trace_retention/_index.md:78]:
```suggestion
If there are specific tags, facets, or groups of traces of which you want to retain more traces than what Intelligent Retention retains, then [create your own retention filter](#create-your-own-retention-filter).
```

File: content/en/tracing/trace_retention/_index.md [hunk 3]
```diff
@@ -16,30 +16,34 @@ further_reading:
...
 
-{{< img src="tracing/trace_indexing_and_ingestion/RetentionFilters.png" style="width:100%;" alt="Retention Filters" >}}
+{{< img src="tracing/trace_indexing_and_ingestion/retention_filters/retention_filter_page.png" style="width:100%;" alt="Retention Filters Page" >}}
 
-In Datadog, on the [Retention Filters tab][3], you can see the following information:
+In Datadog, on the [Retention Filters tab][3], you can see a list of all retention filters:
 
 Filter Name
-: The name of each filter used to index spans. By default, the only filter is [Datadog Intelligent Retention](#datadog-intelligent-retention-filter).
+: The name of each filter used to index spans. By default, the only filter is [Datadog Intelligent Retention Filter](#datadog-intelligent-retention-filter).
```
[the remark this answers is not in the document]
Comment (dussault-antoine) [reply content/en/tracing/trace_retention/_index.md:40]: yep you're correct, this one went under the radar, thanks

Comment (dussault-antoine) [inline content/en/tracing/trace_retention/_index.md:40]:
```suggestion
: The name of each filter used to index spans.
```

File: content/en/tracing/trace_retention/_index.md [hunk 4]
```diff
@@ -16,30 +16,34 @@ further_reading:
...
+By default, two retention filters are enabled for your account:
+- The `Error Default` retention filter indexes error spans with `status:error`. The retention rate and the query can be configured. For instance to only capture production errors, set the query to `status:error, env:production`. Disable the retention filter if you do not want to capture the errors by default.
+- The [Intelligent Retention Filter](#datadog-intelligent-retention-filter) retains diverse error traces and traces from different latency distributions.
 
 You can also create any number of additional [tag-based retention filters](#create-your-own-retention-filter) for your services.
 
 **Note**: Admin rights are required to create, modify, or disable retention filters.
 
-{{< img src="tracing/trace_indexing_and_ingestion/RetentionFilters.png" style="width:100%;" alt="Retention Filters" >}}
+{{< img src="tracing/trace_indexing_and_ingestion/retention_filters/retention_filter_page.png" style="width:100%;" alt="Retention Filters Page" >}}
```
[the remark this answers is not in the document]
Comment (dussault-antoine) [reply content/en/tracing/trace_retention/_index.md:35]: Done

**Review by urseberry — APPROVED** [review]

**Pull request merged**